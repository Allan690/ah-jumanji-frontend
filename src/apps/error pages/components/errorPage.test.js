import {shallow} from "enzyme/build";
import React from "react";
import ErrorPage from "./error404Page";

describe('Error Page test', () => {
  it('should match the snapshot', () => {
    const component = shallow(<ErrorPage debug />);
    expect(component).toMatchSnapshot();
  });
});
