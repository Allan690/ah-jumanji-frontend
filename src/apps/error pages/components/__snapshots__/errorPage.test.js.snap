// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page test should match the snapshot 1`] = `
<Fragment>
  <PageBackground />
  <div
    className="container h-100 d-flex"
  >
    <div
      className="jumbotron-fluid register-redirect-modal my-auto position-relative"
    >
      <em>
        <h1>
          404
        </h1>
      </em>
      <strong>
        <h5>
          Looks Like the page you are looking for was not found
        </h5>
      </strong>
      <withRouter(BackButton) />
    </div>
  </div>
</Fragment>
`;
